Dockerfile

```diff
@@ -42,7 +41,9 @@ RUN dnf -y remove subscription-manager* && \
       postgresql-server-devel \
       qpid-proton-c-devel \
       ruby-devel \
-      wget && \
+      wget \
+      # For seeding ansible runner with ansible-galaxy
+      ansible-5.4.0-3.el8 && \
```

Comment:
Any reason to add this to the end instead of editing the deleted line above?

Reply:
It's mainly because it had the comment attached and I didn't want it to spray that all of it was just for ansible.  Also this was a cherry pick and I did the same thing in the original PR